packages/eos-components/src/components/Carousel.vue

```diff
@@ -71,4 +71,16 @@ export default {
   background-color: $gray-900;
 }
 
+.with-shadow {
+  transition: all ease .4s;
+  box-shadow: $box-shadow-sm !important;
+  &:hover {
+    box-shadow: $box-shadow !important;
+    ::v-deep h3, ::v-deep h4 {
```

Comment:
Using `v-deep` for components we own is not ideal. That's more a workaround for tweaking unmanaged components. I guess you can pass a new boolean property `is-hovered` to the CarouselCard component.